decompile.py

```diff
@@ -53,7 +53,7 @@ def decompile(self, function: str, task_options: Optional[Options] = None) -> De
         pipeline = DecompilerPipeline.from_strings(task_options.getlist("pipeline.cfg_stages"), task_options.getlist("pipeline.ast_stages"))
         task = self._frontend.create_task(function, task_options)
         pipeline.run(task)
-        task.code = self._backend.generate([task])
+        task.code = self._backend.generate([task]) # TODO should we get rid of monkey patching?
```

Comment:
I'm all for it :D
but for now, lets have an issue instead of a random TODO somewhere in a source file

We would probably need to have two different classes for type safety (DecompilerTask and DecompilerResult or something)

Reply:
moved to #170